applications/asset_tracker_v2/src/ext_sensors/ext_sensors.h

```diff
@@ -28,6 +28,11 @@ extern "C" {
 /** @brief Enum containing callback events from library. */
 enum ext_sensor_evt_type {
 	EXT_SENSOR_EVT_ACCELEROMETER_TRIGGER,
+
+	/** Events propagated when an error associated with a sensor device occurs. */
+	EXT_SENSOR_EVT_ACCELEROMETER_DEVICE_ERROR,
+	EXT_SENSOR_EVT_TEMPERATURE_DEVICE_ERROR,
+	EXT_SENSOR_EVT_HUMIDITY_DEVICE_ERROR
```

Comment:
**suggestion**: Those are sensors, not devices.

```suggestion
	EXT_SENSOR_ACCELEROMETER_ERROR_EVT,
	EXT_SENSOR_TEMPERATURE_ERROR_EVT,
	EXT_SENSOR_HUMIDITY_ERROR_EVT
```

Reply:
The library uses the generic sensor API where external driver/hardware is typically referred to as "devices". But I see that this nomenclature probably belongs in the internal `ext_sensors` layer and not at application level (sensor_module). Ill remove the "DEVICE" part.

_Typically we prefix events with the library name with a following `EVT`_ Just to align.